### PR TITLE
Improve logging capture for Pulp Automation

### PR DESCRIPTION
**Pull request by elyezer**
Make sure to use `sudo` when needed and proper set permissions in order to
capture the logs.

### CODE REVIEW

File: ci/jobs/pulp-dev.yaml [hunk 1]
```diff
@@ -89,12 +89,18 @@
...
             if [ "$(which journalctl 2>/dev/null)" ]; then
-                journalctl > logs/syslog
+                sudo journalctl > logs/syslog
             else
-                cat /var/log/messages > logs/syslog
+                if [ -f /var/log/messages ]; then
+                    sudo cat /var/log/messages > logs/syslog
+                else
+                    echo "Failed to get syslogs journalctl neither" \
+                    "/var/log/messages was not found" > logs/syslogs
```
Comment (rbarlow) [inline ci/jobs/pulp-dev.yaml]: How about rewording this to "Failed to get syslogs: neither journalctl nor /var/log/messages were found."

**Comment by rbarlow** [comment]
LGTM!

File: ci/jobs/pulp-dev.yaml (hunk 1 above)
Comment (elyezer) [inline ci/jobs/pulp-dev.yaml]: :+1: much better

**Comment by elyezer** [comment]
I've rebased the PR and addressed the comments. Will merge it.
